### PR TITLE
feature/PPTP-1873 | Remove link to Private Beta by 31/03/22

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Unauthorised Page - Remove link to Private Beta by 31/03/22

https://github.com/hmrc/plastic-packaging-tax-acceptance/pull/119

Screenshot 2022-03-25 at 11 04 02

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

**Comment by ajchilts** [comment]
Do we need to get UCD to verify the page?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html [hunk 1]
```diff
@@ -28,19 +29,27 @@
...
-@contactHmrcLink = {
+@registerLink = {
 @link(
-    id = Some("deskpro_link"),
-    text = messages("unauthorised.paragraph.2.link"),
-    call = Call("GET", config.contactHmrcUrl)
-)
+        id = Some("register_for_ppt_link"),
+        text = messages("unauthorised.paragraph.1.link"),
+        call = Call("GET", StartController.displayStartPage().url)
```
Comment (liampan) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html:36]: StartController.displayStartPage() is a Call I believe?

**Review by liampan — APPROVED** [review]

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html (hunk 1 above)
Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html:36]:
```suggestion
        call = StartController.displayStartPage()
```

Comment (ajchilts) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/unauthorised.scala.html:36]: Yup. It is. I can refactor this. Good spot. 👍

**Review by liampan — APPROVED** [review]

**Pull request merged**